xpk.py

```diff
@@ -4218,10 +4215,12 @@ def get_cpu_env(num_slices, system) -> str:
                   value: "{processes_in_job}"
                 - name: JAX_PROCESS_COUNT
                   value: "{process_count}"
+                {env_vars}
```

Comment:
I guess there's no easy way to do it, but it would be good to assert none of the above env vars are being overwritten here. @RoshaniN Did you hit an error when `JOBSET_NAME` was set twice?

Reply:
Fair observation!
No error, @jonb377 - the YAML generated by XPK showed the JOBSET_NAME (previously) mentioned here and also added by add_env_config . Also, overwriting would not error out.